src/layouts/Settings/FooterSettings.tsx

```diff
@@ -0,0 +1,45 @@
+import { VStack, FormControl, Flex, Switch } from "@chakra-ui/react"
+import { FormLabel, Input } from "@opengovsg/design-system-react"
+import { upperFirst } from "lodash"
+import { useFormContext } from "react-hook-form"
+
+import { Section, SectionHeader } from "layouts/components"
+
+interface FooterSettingsProp {
+  isError: boolean
+}
+
+export const FooterSettings = ({
+  isError,
+}: FooterSettingsProp): JSX.Element => {
+  const { register } = useFormContext()
+
+  return (
+    <Section id="footer-fields">
+      <SectionHeader label="Footer" />
+      <VStack spacing="1.5rem" align="flex-start" w="50%">
+        {["contact", "feedback", "faq"].map((label) => {
+          return (
+            <FormControl isDisabled={isError}>
+              <FormLabel>{upperFirst(label)}</FormLabel>
+              <Input
+                w="100%"
+                id="title"
+                placeholder="https://"
+                {...register(label)}
+              />
```

Comment:
Also, the labels for Contact Us and FAQ don't match exactly because of the keys used - is it possible to do a mapping?

Reply:
actually i got the `https://` from the figma @__@ i could double check with the designers on this! for the second point - i'm not sure i understand exactly what you meant, could you clarify?

Reply:
Ah yup, I saw the figma but just wanted to double check!

For the second point, it's just regarding the field labels not matching what's in the figma exactly - `Faq` not being all caps is more of a nit, but I think having `Contact` instead of `Contact Us` might be a bit confusing for the user